department-of-veterans-affairs/vets-website
# Add Pension home acreage review

## Summary

Adds the question, "What’s the value of the land that’s more than 2 acres?" to the Review page

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/74701

## Testing done

- Start a pension application
- Enter minimal data until you get to the Financial Information > Income and assets > Is your home located on a lot of land that’s more than 2 acres (or 87,120 square feet)?
- Enter YES
- Next question is What’s the value of the land that’s more than 2 acres?
- Enter some $ number
- Proceed with the rest of the application with minimal data
- Look at the review page under the Financial Information page for the question What’s the value of the land that’s more than 2 acres?
- Edit the question "What’s the value of the land that’s more than 2 acres?" to a value that's not a number, and you should see an error
- Edit the question "What’s the value of the land that’s more than 2 acres?" to a number, and you should be able to save the new value without error

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Edit |   N/A     |    <img width="641" alt="Screenshot 2024-01-29 at 4 38 49 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/3cb90de5-816c-4a06-b37d-473e4d93ca6d">   |
| Review | N/A | <img width="680" alt="Screenshot 2024-01-29 at 4 38 57 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/1d3a3068-33b2-4775-b552-7f69777e6eb8"> |

## What areas of the site does it impact?

Pension benefits

## Acceptance criteria

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/74701

### Quality Assurance & Testing


### Error Handling


### Authentication
